assets/src/models/shuttle.ts

```diff
@@ -1,4 +1,48 @@
-import { Vehicle } from "../realtime"
+import { RunId, Vehicle } from "../realtime"
+
+export enum ShuttleVariant {
+  // Rapid Transit Lines
+  Blue = "Blue",
+  Green = "Green",
+  Orange = "Orange",
+  Red = "Red",
+
+  // Other Shuttle Types
+  CommuterRail = "CR",
+  Special = "Special",
+}
+
+export const shuttleVariantFromRunId = (
+  runId: RunId
+): ShuttleVariant | null => {
+  switch (runId) {
+    case "999-0501":
+      return ShuttleVariant.Blue
+    case "999-0502":
+      return ShuttleVariant.Green
+    case "999-0503":
+      return ShuttleVariant.Orange
+    case "999-0504":
+      return ShuttleVariant.Red
+    case "999-0505":
+      return ShuttleVariant.CommuterRail
+    case "999-0555":
+      return ShuttleVariant.Special
+    default:
+      return null
+  }
+}
+
+const prefix = (variant: ShuttleVariant | null): string => {
+  switch (variant) {
+    case null:
+      return ""
+    case ShuttleVariant.CommuterRail:
+      return "Commuter Rail "
```

Comment:
y'know, I'm not certain I did this "right" anyway! I don't think we should be relying on the underlying value of the enum anyway.

I actually started in https://github.com/mbta/skate/blob/main/assets/src/components/shuttlePicker.tsx#L61-L64 when I started looking for existing shuttle code, and that was before I found https://github.com/mbta/skate/blob/main/assets/src/models/shuttle.ts .

So I copied `CR` from `shuttlePicker.tsx`. I think that because this is a prefix, it doesn't belong in a "global" enum, so I'll go ahead and make a change and see what you think

Reply:
That works! In a way, the former setup was more succinct, but I think this is correct and readable, so I have no problem with the new approach 👍